cip/2.testable/CIP2015-06-24-call-procedures.adoc

```diff
@@ -69,21 +69,47 @@ The details of this are outside the scope of this proposal.
 
 === Procedure calls
 
-Procedures are called for each incoming record produced by preceding clauses and each time produce arbitrarily many result records. If a statement starts with a procedure call, this call is executed once, i.e. procedure calls behave similiarly to `UNWIND` regarding how they change the result cardinality.
+Procedures are called for each incoming record produced by preceding clauses and each time produce arbitrarily many result records. If a statement starts with a procedure call, this call is executed once, i.e. procedure calls behave similarly to `UNWIND` regarding how they change the result cardinality.
 
 We propose new syntax that introduces a new clause for calling procedures.
 This clause for calling procedures starts with the `CALL` keyword.
 Next follows the actual call (i.e. procedure name and arguments in signature order).
 Finally (and optionally) the clause may end with the `YIELD` keyword followed by any chosen result field projections.
 
-A procedure call constitutes either an entire Cypher statement (i.e. is a "standalone call"), or is called as part of a larger query (i.e. is an "in-query call"). In standalone calls, arguments may be either passed implicitly or explicitly. These different calling modes are detailed in the following subsections.
+A procedure call constitutes either an entire Cypher statement (i.e. is a "standalone call"), or is called as part of a larger query (i.e. is an "in-query call"). These different calling modes are detailed in the following subsections.
+
+When `CALL` is used within a query, `YIELD` may be followed by a `WHERE` subclause.
+This `WHERE` subclause applies _after_ the invocation of the procedure, even in the case where all the variables referenced in the `WHERE` subclause originate from before the `CALL`.
+
+Considering the procedure `querySQL(dbURI``::``STRING?, query``::``STRING?)``::`` (row``::``MAP)`:
+
+.Filtering the result from a procedure
+[source, cypher]
+----
+CALL querySQL("jdbc:mysql://localhost:3306/foo","SELECT bar FROM baz")
+YIELD row
+WHERE row.bar > "quux"
```

Comment:
Style guide suggests indenting subclauses with two spaces (`WHERE` and `YIELD`).